src/ssl.c

```diff
@@ -15763,8 +15768,6 @@ int wolfSSL_SetSession(WOLFSSL* ssl, WOLFSSL_SESSION* session)
         SESSION_ROW_UNLOCK(sessRow);
         sessRow = NULL;
     }
-    /* Make sure we don't access this anymore */
-    session = NULL;
```

Comment:
I added this to make sure we don't slip up in the future and try to access this var after this point. Maybe at least leave a nice big comment that `session` can't be accessed after this point?

Reply:
Sounds good! I'll put a big comment instead.